scalafix-reflect/src/main/scala-3/scalafix/internal/reflect/RuleCompiler.scala

```diff
@@ -0,0 +1,88 @@
+package scalafix.internal.reflect
```

Comment:
this is a first stub, disregard as it needs to be reimplemented with the new scala3 compiler APIs